sorting/merge_sort.cpp

```diff
@@ -30,44 +30,28 @@
  *
  * @param l - end index or right index of second half array
  */
-void merge(int *arr, int l, int m, int r) {
+void merge(int &arr, int l, int m, int r) {
```

Comment:
use reference instead of pointer.

Reply:
![image](https://user-images.githubusercontent.com/7001608/90171410-76a73780-dd6f-11ea-8f36-701532973140.png)
You can review all the CI errors and the descriptions [here](https://github.com/TheAlgorithms/C-Plus-Plus/pull/1017/checks?check_run_id=981737416).